Automattic/wp-calypso
# Checkout: Hide check out help link and rely on masterbar

Fixes https://github.com/Automattic/dotcom-forge/issues/3231

## Proposed Changes

* Hide checkout help link in checkout and rely on help center in masterbar

## Testing Instructions

* View a domain only transfer checkout via http://calypso.localhost:3000/setup/domain-transfer/intro
* The checkout should no longer link to plan information
* Do the same for checkout of other products.

Before

![Screenshot 2023-07-27 at 15-48-54 Checkout — WordPress com](https://github.com/Automattic/wp-calypso/assets/811776/3e4a92d7-1177-48ca-8299-a8440298ddc6)


After

![Screenshot 2023-07-27 at 15-42-59 Checkout — WordPress com](https://github.com/Automattic/wp-calypso/assets/811776/ab0debae-47c4-4d4d-89ca-b4bcfb2f3fc8)
